integration/app/test-demo/src/pages/Home.tsx

```diff
@@ -19,6 +19,16 @@ export const Home: React.FC = () => (
           'useAttendeeAudioStatus',
         ]}
       />
+      <TestApp
```

Comment:
![image](https://user-images.githubusercontent.com/20471438/155632973-0da65797-8c01-432c-aae5-22e3212e949a.png)
The layout feel like a little crowded. Some `margin-bottom` should help. 
No worroy, I will raise a PR to improve the `TestApp` Component.

Reply:
done

Reply:
Thank you!